doc/languages-frameworks/r.section.md

```diff
@@ -94,6 +94,43 @@ Executing `nix-shell` will then drop you into an environment equivalent to the
 one above. If you need additional packages just add them to the list and
 re-enter the shell.
 
+## torch with GPU acceleration {#torch-gpu-accel}
+
+The `torch` package included in `nixpkgs` is based on the binary packages by the
+mlverse team. By default, the CPU version of the package is installed. To enable
+GPU acceleration, set `config.cudaSupport` to true when importing `nixpkgs` in a
+`shell.nix`. You will also need to use `nixglhost`. Here is an example of such a
```

Comment:
My understanding is that you only need `nixglhost` if you are not on NixOS.

Reply:
This should be the case, but I don't see any patching happening in this PR so likely it won't work on NixOS either. Should be possible to patch the binaries to operate without tricks on NixOs.

Reply:
Yes, it is possible to patch the liblantern and libtorch binaries, I did it successfully in #273536. After patching, the binaries ran on both NixOS and nix in CentOS. I didn't try patching the full bundle being installed in this PR though.

Whether `nixglhost` works on NixOS is another thing to test.

Reply:
@Kupac tested on NixOS and confirmed he needed `nixglhost` as well. @Kupac could you perchance leave a review and describe the process? Is the description from the documentation aligned with what you had to do?

Reply:
> tested on NixOS and confirmed he needed nixglhost as well.

This is a bug then, the appropriate patchelfing is necessary. Nixglhost should not be required on NixOS

Reply:
I have amended the docs to explain this is only for non-NixOS linux and that we are looking for people to fix torch for NixOS and darwin.